### PR TITLE
feat: update eslint

### PR DESCRIPTION
**Pull request by shamsartem**

### CODE REVIEW

File: src/lib/configs/project/fluence.ts [hunk 1]
```diff
@@ -984,7 +984,7 @@ ${yamlDiffPatch(
 # customFluenceEnv:
 #   contractsEnv: local
 #   relays:
-#     - "${kras[0]?.multiaddr}"
+#     - "/ip4/127.0.0.1/tcp/10010/ws/p2p/12D3KooWGpqaLpVUoLB5cJPLDS6CjBNZePBNTYPGpNPNZMDs6osY"
```
Comment (folex) [inline src/lib/configs/project/fluence.ts:987]: why?

File: src/lib/helpers/utils.ts [hunk 2]
```diff
@@ -66,7 +68,12 @@ export function jsonStringify(unknown: unknown): string {
 
 export function stringifyUnknown(unknown: unknown): string {
```
Comment (folex) [inline src/lib/helpers/utils.ts:69]: Could you please add a comment on what's the use-case for this function and what cases this function covers. 

Seems like it's error oriented, maybe call it correspondingly?

**Review by folex — APPROVED** [review]
Awesome!

File: src/lib/configs/project/fluence.ts (hunk 1 above)
Comment (shamsartem) [reply src/lib/configs/project/fluence.ts:987]: this is just an example comment anyway that most people will not ever see cause we turned off comments in configs by default. It's just simpler to have an example that doesn't unnecesserally rely on anything concrete (such as here where kras[0] can be undefined according to typescript even though we know it never is)

File: src/lib/helpers/utils.ts (hunk 2 above)
Comment (shamsartem) [reply src/lib/helpers/utils.ts:69]: yeah I mostly use this with errors cause one can throw anything in js (not only errors) and it's good to be able to represent the error in a string form. but I also used it for e.g. debug logs or "unreachable error" messages where we don't neceserraly care much about the output as long as it's somewhat readable. Maybe would be good to have two functions for this each with a more clear purpose for existance and used accordingly. Added task to do that https://linear.app/fluence/issue/DXJ-763/refactor-stringifyunknown-function

**Pull request merged**